### PR TITLE
Add spam and phishing domains (issues #2509, #2510)

### LINKED ISSUE #2509: Spam-Domänen
Hallo,

hier wieder eine Sammlung an Spam-Domänen:

smart-assets-io-3719.s3.amazonaws.com
routeslim-de.itemdb.com
mutsce.com
robiograms.com
ratheast.com
casinosbonuss.net
itemdb.com
mta-as-path.itemdb.com
cliszs.accountcassette.com
soslos.accountcassette.com
owlos.accountcassette.com
accountcassette.com
kalwl.accountcassette.com
cdfvb.accountcassette.com
dojhq.accountcassette.com
365proplus.site
cliszs.accountcassette.com
soslos.accountcassette.com
skglw.accountcassette.com
tyqsb.accountcassette.com
byabj.olynatr-gnomem5565-99.xyz
olynatr-gnomem5565-99.xyz
royal-cloud-wi1qq.apitest-3ujr5vljnp.workers.dev
apitest-3ujr5vljnp.workers.dev
mail.loonesryfs.com
pg879.jashokin.com
ofh012.jashokin.com
ud187.jashokin.com
sn96.jashokin.com
ul974.jashokin.com
6x1h3ip.mindoasis.click
ib970.edevelop.space
edevelop.space
xt88.edevelop.space
zo08.edevelop.space
ka14.playtotriumph.com
xc564.playtotriumph.com
ql955.playtotriumph.com
xkcu6lywzuu7jxb.hillsidenow.com
hillsidenow.com
mp4jbnjph0.purevbe.top
pje559.myrule.space
jr92.myrule.space
myrule.space
cis485.myrule.space
ywu331.myrule.space
mc21.myrule.space
rsd9arcdmr1.blogfreecookbook.com
blogfreecookbook.com
0y5xfn19j0xth.primeessence.lol
tsp743.derentyls.com
bp71.derentyls.com
vi93.derentyls.com
wpy95.derentyls.com
mochis.tecnm.mx
brc8y6k.treehousefamilyadvisory.com
treehousefamilyadvisory.com
osz20.tenfas.com
7idofkwwv40xm.aeromind.garden
uq17.tenfas.com
awa49.tenfas.com
e6yo1f2rs4mkh.novaspirit.casa
shn82.myloves.space
gqi56.myloves.space
myloves.space
ptx98.myloves.space
ce78.myloves.space
cu93.myloves.space
rei89.kasiobask.com
dsp028.kasiobask.com
qw453.kasiobask.com
stg708.kasiobask.com
rh738.kasiobask.com
lee9ytn4v0.inmycorner.dev
inmycorner.dev
0okgo54ur.purevbe.top
jph07.gamewizen.com
zzu690.gamewizen.com
mva95.gamewizen.com
benefits.more2flirt.com
more2flirt.com
specials.more2flirt.com
www.more2flirt.com
unsubscribe.more2flirt.com
cityworking.com.br
0xuytir.markwjohnson.org
markwjohnson.org
co80.mytrex.space
bhp024.mytrex.space
mytrex.space
yl92.mytrex.space
km773.mytrex.space
jc220.mytrex.space
Hymer-auswahlrunde.de
artistioggi.org
9ehmgg6y053nf5.regalbars.co.uk
regalbars.co.uk
rfysohf2gb.deepcalm.hair
xtt210.cmpssc.com
cmpssc.com
pr90.cmpssc.com
aiu69.cmpssc.com
kie69.cmpssc.com
yaa64.cmpssc.com
DeLonghi-kundenservice.de
willowhousing.org.uk
a3wwowq.ethnobalance.beer
la75.wisey.space
sbm89.wisey.space
wisey.space
vll167.wisey.space
vqb72.wisey.space
hh729.wisey.space
hw71.wisey.space
ury65.nexoexpresses.com
mki335.nexoexpresses.com
ys147.nexoexpresses.com
leb253.nexoexpresses.com
ko67.nexoexpresses.com
e4rv750gsv.primeessence.lol
zxg636.catholicbali.com
yj204.catholicbali.com
pp680.catholicbali.com
jm014.catholicbali.com
zk365.topbest.space
topbest.space
zij588.topbest.space
sls605.topbest.space
wsr70.topbest.space
zt174.topbest.space
fgf96.topbest.space
tl19.topbest.space
d613kcbv8wga.photobooth.software
photobooth.software
amicacs.co.ke
4tpjg412s04s.photobooth.software
na3bribnu.olynatr-bet-velo3d-85.xyz
olynatr-bet-velo3d-85.xyz
vet.just.edu.jo
2gag3frqvb4a.glavinus.beer
glavinus.beer
jaf789.madeof.space
madeof.space
ad52.madeof.space
ad870.madeof.space
lfz910.madeof.space
dk858.madeof.space
mxr68.madeof.space
cwq747.chillj.com
xhh971.chillj.com
eht435.chillj.com
d1m6ved6xi.slowliving.beauty
tx091.playcentrals.com
sn276.playcentrals.com
www.fremdgehen-mail.com
register.fremdgehen-mail.com
qgt12.playtotriumph.com
us00.playtotriumph.com
lo07.playtotriumph.com
zz06.playtotriumph.com
ts403.priney.space
priney.space
bft14.priney.space
smx08.priney.space
dly53.priney.space
vzv18.priney.space
cliszs.affecthurl.com
soslos.affecthurl.com
owlos.affecthurl.com
affecthurl.com
kalwl.affecthurl.com
cdfvb.affecthurl.com
dojhq.affecthurl.com
cliszs.affecthurl.com
soslos.affecthurl.com
skglw.affecthurl.com
tyqsb.affecthurl.com
juachigranover.com
bondence.com
imperiodized.com
virtuarters.com
buyto.shop
ueh.edu.vn
auto-notfallset.shop
ksxbg.accountcassette.com
ofbkl.accountcassette.com
ksxbg.affecthurl.com
ofbkl.affecthurl.com
unrealsoft.bg
vdartinc.com
techbazaronline.co.uk
dkb-auth.de
sevki.accountcassette.com
tgavi.accountcassette.com
sevki.affecthurl.com
tgavi.affecthurl.com
dnw2-718340643432.us-central1.run.app
thestitchingparlorinc.com
vr-online-konto-aktualisierung.cfd
awl-zentrum.de
serenity-mail.de
lustich.de
Lustich.de-index.info

### LINKED ISSUE #2510: Phishing-Domänen
vdartinc.com
techbazaronline.co.uk
dkb-auth.de
siddahmed.com
thestitchingparlorinc.com
vr-online-konto-aktualisierung.cfd
net-compliance-support.live
saktualisierung106.online
tawin.co
saspk.info

### PR DESCRIPTION
**Pull request by AincradBot**
Adds the reported spam and phishing domains to the corresponding blocklists.

**Changes**
- `Blocklisten/spam.mails`: +196 domains from issue #2509
- `Blocklisten/Phishing-Angriffe`: +10 domains from issue #2510

**Details**
- Entries use the repository's ABP format (`||domain^`), inserted at the correct alphabetical position, line endings (CRLF) preserved.
- No existing lines were modified (206 insertions, 0 deletions).
- 3 duplicates already present in `spam.mails` were skipped (`artistioggi.org`, `virtuarters.com`, `vdartinc.com`); `vdartinc.com` is still added to `Phishing-Angriffe` per issue #2510.
- Domains were normalized to lowercase for consistency (e.g. `Hymer-auswahlrunde.de`, `Lustich.de-index.info`).
- Verified: no internal duplicates, all 206 new entries present exactly once.

Fixes #2509
Fixes #2510

---
*PR erstellt von einem automatisierten Agent (AincradBot). Inhalt geprüft gegen die Angaben in den Issues; die fachliche Freigabe der Domains liegt beim Repo-Maintainer.*

### CODE REVIEW

**Comment by RPiList** [comment]
Domain Input has to be checked by our software. So no automated input possible. Sorry. Thanks for supporting our project. Greetings.
